docs/build_version_doc/update_all_version.sh

```diff
@@ -68,26 +73,54 @@ fi
 # Write all version numbers into $tag_file for AddVersion.py to use later
 # Master is added by that script by default
 for tag in $tag_list; do
-    if [ $tag != 'master' ]
-    then
-        echo "$tag" >> "$tag_file"
-    fi
+    echo "$tag" >> "$tag_file"
 done
 
+function update_fixes {
+  tag=$1
+  echo "Begin update fixes.."
+  # All fixes are done on the master branch of mxnet-incubator repository
+  # During a nightly build, these fixes will be patched to all the versions in the asf-site repository including the master folder under versions directory.
+  # copy <master folder location> <version folder location>
+  
+  #SIM: https://sim.amazon.com/issues/AWSDBUX-19727
+  echo "Copying mxnet.css from master branch to all versions...."
+  cp "$MASTER_SOURCE_DIR/$STATIC_FILES_DIR/mxnet.css"  "$built/versions/$tag/_static"
+
+  echo "Update fixes complete.."
+}
+
 # Update the specified tags with the Versions dropdown
 for tag in $tag_list; do
     # This Python script is expecting the tag_list.txt and it will use that as the entries to populate
+
+  #SIM: https://sim.amazon.com/issues/AWSDBUX-19727
+  if [[ $tag == '1.1.0' || $tag == 'master' ]]
+  then
+     echo "Copying install/index.html from master branch file to all versions...."
+     cp "$MASTER_SOURCE_DIR/$BUILD_HTML_DIR/install/index.html"  "$built/versions/$tag/install"
+  fi
+
     python AddVersion.py --root_url "$root_url" --file_path "$built/versions/$tag" --current_version "$tag" || exit 1
 
-    if [ $tag != 'master' ]
+    if [ $tag == 'master' ]
```

Comment:
I don't think this is correct. Why would you not want the different packages available in each version?